TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_command_runner.py

```diff
@@ -58,3 +60,40 @@ def test_command_runner_sanitizing_a_flag_field_is_a_noop():
     }, sensitive_fields=["--use-password"])
     assert runner.command == ["command", "--username", "admin", "--use-password"]
     assert runner.sanitized_command() == ["command", "--username", "admin", "--use-password"]
+
+
+def test_command_runner_prints_output_as_run_default(capsys, mocker):
+    mock_stdout = "Found 5 directories"
+    mock_stderr = "Unknown file path"
+    mock_subprocess_result = SimpleNamespace(stdout=mock_stdout, stderr=mock_stderr)
```

Comment:
huh, this is new to me. It's basically an empty object that you can just add attributes to?

Reply:
Yeah essentially, I've found it useful for testing complex objects that I want to simulate. See my other comment but I'm hoping I can replace with the actual object